.github/workflows/go.yml

```diff
@@ -149,6 +149,13 @@ jobs:
         run: go mod download
         working-directory: src
 
+      - name: Install GoReleaser Pro
+        uses: goreleaser/goreleaser-action@v6
+        with:
+          install-only: true
+        env:
+          GORELEASER_KEY: ${{ secrets.GORELEASER_KEY }}
+
       - name: Run GoReleaser (Linux)
         uses: goreleaser/goreleaser-action@v6
```

Comment:
wouldn't this step still install `goreleaser` and cause the same issue as before? My understanding is that `install-only` just makes the `goreleaser` binary available so subsequent steps can simply run that binary. That's what I gather from their example: https://github.com/goreleaser/goreleaser-action?tab=readme-ov-file#install-only

Reply:
Also, you'll want to make sure it installs pro.